### PR TITLE
Notebook: use UDTF to normalize a table column

### PR DESCRIPTION
**Pull request by pearu**
As in title.

### CODE REVIEW

File: notebooks/rbc-omnisci-udtf-normalize.ipynb [hunk 1]
```diff
@@ -0,0 +1,338 @@
+{
```
Comment (guilhermeleobas) [inline notebooks/rbc-omnisci-udtf-normalize.ipynb:1]: <p>Maybe replace the last comment by: <code>return the size of output column</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xnd-project/rbc/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (guilhermeleobas) [inline notebooks/rbc-omnisci-udtf-normalize.ipynb:1]: <p>Same here!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xnd-project/rbc/pull/164/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (pearu) [reply notebooks/rbc-omnisci-udtf-normalize.ipynb:1]: done, thanks!

**Comment by guilhermeleobas** [comment]
Build is failing in the conda test (omniscidb 5.2). Do you know if `load_table_columnar` works in older versions?

**Comment by pearu** [comment]
It should work. However, the failing test does not use load_table_columnar. I suspect that the recent changes to thrift config might cause the failure. I'll try to reproduce locally..

**Pull request merged**